File: BehavioralDesignPattern/ObserverPattern/java/Driver.java
package BehavioralDesignPattern.ObserverPattern.java;

import BehavioralDesignPattern.ObserverPattern.java.Observer.Observer;
import BehavioralDesignPattern.ObserverPattern.java.Subject.NewsArticle;

public class Driver {
    public static void main(String[] args){
        NewsArticle newsArticle = new NewsArticle();

        Observer o1 = new Observer("Ankit");
        Observer o2 = new Observer("Taylor");
        Observer o3 = new Observer("Tanmoy");

        newsArticle.attach(o1);
        newsArticle.attach(o2);
        newsArticle.attach(o3);

        newsArticle.publishArticle("This is the new Article");
    }
}

File: BehavioralDesignPattern/ObserverPattern/python/Driver.py
from Observer.Observer import Observer
from Subject.NewsArticle import NewsArticle


if __name__ == "__main__":
    newsarticle = NewsArticle()
    
    o1 = Observer("Ankit")
    o2 = Observer("Taylor")
    o3 = Observer("Tanmoy")

    newsarticle.attach(o1)
    newsarticle.attach(o2)
    newsarticle.attach(o3)

    newsarticle.publishArticle("Welcome to Observer Pattern")